src/Test/Utilities/Shared/Diagnostics/DiagnosticExtensions.cs

```diff
@@ -199,9 +199,12 @@ public static TCompilation VerifyDiagnostics<TCompilation>(this TCompilation c,
             }
 
             Compilation newCompilation;
-            var driver = AnalyzerDriver.CreateAndAttachToCompilation(c, analyzersArray, options, AnalyzerManager.Instance, onAnalyzerException, null, false, out newCompilation, CancellationToken.None);
-            var discarded = newCompilation.GetDiagnostics();
-            diagnostics = driver.GetDiagnosticsAsync(newCompilation).Result.AddRange(exceptionDiagnostics);
+            using (var context = AnalyzerHostContext.Create())
+            {
+                var driver = AnalyzerDriver.CreateAndAttachToCompilation(c, analyzersArray, options, (DefaultAnalyzerHostContext)context, onAnalyzerException, null, false, out newCompilation, CancellationToken.None);
```

Comment:
that change can come later I believe when we figure out what to do with all reference to DefaultAnalyzerHostContext in compiler code. ideally, all should just point to AnalyzerHostContext rather than concrete type.